### ISSUE TITLE
Steer away from a hard-coded approach

### ISSUE THREAD

**Issue by B3none** [nibalizer/case-watch-bot#9]
Scraping the various sites to get the data is fine, however, I would avoid using very specific CSS queries to find the correct element. 

It will no-doubt work for the meantime but if there's a way you can dynamically find the element that'd probably be better as classes like this could potentially change with a new website deployment.

https://github.com/nibalizer/case-watch-bot/blob/master/main.js#L160

**Comment by B3none** [comment]
Perhaps you could look at parsing and dynamically generating commands based on this website.

https://www.arcgis.com/apps/opsdashboard/index.html#/bda7594740fd40299423467b48e9ecf6

**Comment by nibalizer** [comment]
Right! We've had a lot of churn as the various state's change their designs all the time. Fortunately, jquery/cheerio is quick to develop with so it usually doesn't take much time to get them fixed. The bot will squak in discord when it sees an update so it's really easy to see when a state has changed for parsing.

I believe that visualization comes from [this data](  https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data)

We could use that data I suppose but other discord bots already do [that]( https://top.gg/bot/683462722368700526 )

So I sorta think webscraping is the way to differentiate?

**Comment by B3none** [comment]
> Right! We've had a lot of churn as the various state's change their designs all the time. Fortunately, jquery/cheerio is quick to develop with so it usually doesn't take much time to get them fixed. The bot will squak in discord when it sees an update so it's really easy to see when a state has changed for parsing.
> 
> I believe that visualization comes from [this data](https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data)
> 
> We could use that data I suppose but other discord bots already do [that](https://top.gg/bot/683462722368700526)
> 
> So I sorta think webscraping is the way to differentiate?

You're right, how about we use the CSV data you've linked as a fallback and prioritise the scraping method? It gives you plenty of coverage but also gives you the ability to override any location.

That way we have the option to differentiate but we're also not shooting ourselves in the foot just to offer an alternative solution. What're your thoughts on this approach?

**Comment by nibalizer** [comment]
Sounds great! In theory the state data is more up to date but the breadth of the CSV data is also very valuable.
